cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/Argument.java

```diff
@@ -22,6 +23,7 @@
 
     private boolean privateArgument;
     private Prompt prompt;
+    private boolean isExpression;
```

Comment:
use `expression` instead of `isExpressions`. The 'is' should appear only in getters/setters. Fields should be named without 'is'